src/theme/styles/colours.ts

```diff
@@ -60,6 +61,8 @@ export const BASE_COLOURS = {
   red1: '#FF305B',
   red2: '#FF6871',
   red3: '#F82D3A',
+  red4: '#F13E6E',
+  green: '#41C29B',
```

Comment:
If you are adding `red4`, do the same for `green` and add `green4`.

But actually, why not to use the existing reds and greens?
From the designs the colors should be the same as the ones already in place.

Reply:
@alfetopito I've checked the colors, but no existing one was matching to the design. So I've added new one.

Reply:
I see. No worries, just pick of of the existing colors, I don't think we need to introduce new green/red for this component.

Reply:
cc @alongoni